### PR TITLE
Add documentation translation

### PR DESCRIPTION
**Pull request by TristeOak**
Changelog
---------

### Added

- Add Chinese translation of configuration.md and intro.mdx

### Security

Checklist
---------

- [x] Test
- [ ] Self-review
- [x] Documentation

### CODE REVIEW

**Comment by QubitPi** [comment]
Thanks for contributing, @TristeOak .

Meanwhile, please rebase your fork to fix the CI/CD error above (We've patched our `master` branch in order to fix that error). When that get's done, I'll take a look at this today.

If you are interested in joining us for an internship, please ping us on zhipin.com and mention this PR

If this PR gets merged, we will add you to our contributor list displayed on our README

Thanks again for contributing.

**Comment by TristeOak** [comment]
Hi Jiaqi,

My name is Ziyao Jin. I'm very interested in open source projects but I'm just a beginner. I'm currently studying at ANU. Since the summer break in Australia is relatively long, I really want to do an internship to further study software engineering. 

Please let me know if my translation is accurate, as I'm still somewhat unclear about some concepts. I'm eager to contribute more to open source projects and I will continue to look for issues that I can solve on GitHub. 

I‘ve rebased the fork and I will contact you on Zhipin after you review my work to check if it meets the standards.

Sincerely,
Ziyao Jin

File: docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/configuration.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+| MAVEN_SETTINGS_XML             | Webservice 项目的 [Maven 设置文件](https://maven.apache.org/settings.html)                                                                                         | 包含[这些元标签](https://github.com/QubitPi/jersey-webservice-template/blob/jpa-elide/settings.xml.example)的 `settings.xml` 文件的确切内容                                                            |
+| APPLICATION_PROPERTIES         | 上述提到的 `src/main/resources/application.properties` 文件的内容                                                                                                    | 请参阅上述的 [Core Properties](#core-properties) 部分                                                                                                                                                              |
+| OAUTH_PROPERTIES               | 上述提到的 `src/main/resources/oauth.properties` 文件的内容                                                                                                          | 请参阅上述的 [Security](#security) 部分                                                                                                                                                                            |
```
Comment (QubitPi) [inline docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/configuration.md:97]: `#security` -> `##oauth-2`

File: docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/configuration.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+| JPADATASTORE_PROPERTIES        | 上述提到的 `src/main/resources/jpadatastore.properties` 文件的内容                                                                                                   | 请参阅上述的 [JPA DataStore](#jpa-datastore) 部分                                                                                                                                                                  |
```
Comment (QubitPi) [inline docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/configuration.md:98]: `#jpa-datastore` -> `#elide-jpa-datastore`

File: docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/configuration.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+| DATA_MODELS_PRIVATE_REPO_TOKEN | 用于 Elide 数据模型存储库的 GitHub Fine-grained token，至少具有“对代码和元数据的读取权限”                                                | [创建 fine-grained personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-fine-grained-personal-access-token) |
```
Comment (QubitPi) [inline docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/configuration.md:99]: `用于 Elide 数据模型存储库的 GitHub Fine-grained token` -> `用于访问 Elide Data Model 代码仓库的 GitHub PAT token`

File: docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/configuration.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+| DATA_MODELS_PRIVATE_REPO_ORG   | Elide 数据模型的 GitHub 存储库的组织或用户名称                                                                                                                         | 对[这个例子](https://github.com/QubitPi/jersey-webservice-template-jpa-data-models)来说，`DATA_MODELS_PRIVATE_REPO_ORG` 的值为 "QubitPi"                                                                               |
```
Comment (QubitPi) [inline docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/configuration.md:100]: - `Elide Data Model 代码仓库`
- `组织` -> `GitHub 组织`

File: docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/configuration.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+| DATA_MODELS_PRIVATE_REPO_NAME  | Elide 数据模型的 GitHub 存储库的名称                                                                                                                                  | 对[这个例子](https://github.com/QubitPi/jersey-webservice-template-jpa-data-models)来说，`DATA_MODELS_PRIVATE_REPO_NAME` 的值为 "jersey-webservice-template"                                                           |
```
Comment (QubitPi) [inline docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/configuration.md:101]: `Elide Data Model 代码仓库`

File: docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/configuration.md [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+| AWS_ACCESS_KEY_ID              | 与[配置 AWS CLI 的环境变量](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html)中提到的完全相同的 `AWS_ACCESS_KEY_ID`     | [如何为 Amazon Keyspaces 创建和配置 AWS 凭据](https://docs.aws.amazon.com/keyspaces/latest/devguide/access.credentials.html)                                                                  |
```
Comment (QubitPi) [inline docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/configuration.md:102]: `AWS 凭据` -> `AWS 访问密钥 ID`

File: docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/configuration.md [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+| AWS_SECRET_ACCESS_KEY          | 与[配置 AWS CLI 的环境变量](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html)中提到的完全相同的 `AWS_SECRET_ACCESS_KEY` | [如何为 Amazon Keyspaces 创建和配置 AWS 凭据](https://docs.aws.amazon.com/keyspaces/latest/devguide/access.credentials.html)                                                                  |
```
Comment (QubitPi) [inline docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/configuration.md:103]: `AWS 凭据` -> `AWS 访问密钥 ID`

File: docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/configuration.md [hunk 8]
```diff
@@ -0,0 +1,112 @@
...
+| AWS_REGION                     | 与[配置 AWS CLI 的环境变量](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html)中提到的完全相同的 `AWS_REGION`            | [如何为 Amazon Keyspaces 创建和配置 AWS 凭据](https://docs.aws.amazon.com/keyspaces/latest/devguide/access.credentials.html)                                                                  |
```
Comment (QubitPi) [inline docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/configuration.md:104]: `AWS 凭据` -> `AWS 访问密钥 ID`

File: docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/configuration.md [hunk 9]
```diff
@@ -0,0 +1,112 @@
...
+| AWS_WS_TFVARS                  | HashiCorp Terraform 变量 [值文件](https://qubitpi.github.io/hashicorp-terraform/terraform/language/values/variables#variable-definitions-tfvars-files) 内容       | [hashicorp-aws](https://qubitpi.github.io/hashicorp-aws/docs/webservice)                                                                                                                                           |
+| MAVEN_SETTINGS_XML             | Webservice 项目的 [Maven 设置文件](https://maven.apache.org/settings.html)                                                                                         | 包含[这些元标签](https://github.com/QubitPi/jersey-webservice-template/blob/jpa-elide/settings.xml.example)的 `settings.xml` 文件的确切内容                                                            |
+| APPLICATION_PROPERTIES         | 上述提到的 `src/main/resources/application.properties` 文件的内容                                                                                                    | 请参阅上述的 [Core Properties](#core-properties) 部分                                                                                                                                                              |
```
Comment (QubitPi) [inline docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/configuration.md:96]: `#core-properties` -> `#主要配置`

File: docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/intro.mdx [hunk 10]
```diff
@@ -0,0 +1,296 @@
...
+[//]: # (distributed under the License is distributed on an "AS IS" BASIS,)
+[//]: # (WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.)
+[//]: # (See the License for the specific language governing permissions and)
+[//]: # (limitations under the License.)
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+想要一个API吗？
+-------------------
```
Comment (QubitPi) [inline docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/intro.mdx:24]:
```
需要搭建一个 API ？
----------------
```

File: docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/intro.mdx [hunk 11]
```diff
@@ -0,0 +1,296 @@
...
+
+:::info 请确保已经安装 JDK 17, Maven, 和 Docker Engine 🤗
+
+- 我们提供如何安装 JDK 17 和 Maven 的[教程](setup#installing-java--maven-on-mac)
+- 我们也提供[安装 Docker Engine 的链接](setup#installing-docker-engine)
+
+:::
+
+将模版实例化
+--------------------------
```
Comment (QubitPi) [inline docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/intro.mdx:34]:
```
将仓库模版实例化
--------------
```

File: docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/intro.mdx [hunk 12]
```diff
@@ -0,0 +1,296 @@
...
+
+请访问[Astraios GitHub]，然后执行以下操作之一：
+
+1. 使用 `git clone https://github.com/paion-data/astraios.git` 克隆存储库，或者
```
Comment (QubitPi) [inline docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/intro.mdx:38]: `存储库` -> `代码仓库`

File: docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/intro.mdx [hunk 13]
```diff
@@ -0,0 +1,296 @@
...
+2. 用我们自己的 Web 服务名称执行以下操作来将模版实例化：
+
+  ![Error loading instantiation-1.png](./img/instantiation-1.png)
+  ![Error loading instantiation-2.png](./img/instantiation-2.png)
+
+创建模型
+---------------
```
Comment (QubitPi) [inline docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/intro.mdx:45]:
```
创建数据模型
----------
```

File: docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/intro.mdx [hunk 14]
```diff
@@ -0,0 +1,296 @@
...
+```java
+import com.yahoo.elide.annotation.Include;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "Book")
```
Comment (QubitPi) [inline docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/intro.mdx:61]: `Book` -> `book`

[JPA entity is better off using non-caped table name](https://dev.mysql.com/doc/refman/8.0/en/identifier-case-sensitivity.html#:~:text=Table%20names%20are%20stored%20in,database%20names%20and%20table%20aliases.). I just noticed this. Not your fault.

File: docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/intro.mdx [hunk 15]
```diff
@@ -0,0 +1,296 @@
...
+```bash
+git clone https://github.com/QubitPi/jersey-webservice-template-jpa-data-models.git
+cd jersey-webservice-template-jpa-data-models
+mvn clean install
+```
+
+:::
+
+加载数据模型
+-------------------
```
Comment (QubitPi) [inline docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/intro.mdx:95]:
```
加载数据模型
----------
```

File: docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/intro.mdx [hunk 16]
```diff
@@ -0,0 +1,296 @@
...
+    </profiles>
+
+    <activeProfiles>
+        <activeProfile>astraios-data-models</activeProfile>
+    </activeProfiles>
+</settings>
+```
+
+运行
+-------
```
Comment (QubitPi) [inline docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/intro.mdx:125]:
```
运行
----
```

File: docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/intro.mdx [hunk 17]
```diff
@@ -0,0 +1,296 @@
...
+
+有了定义的数据模型之后，我们可以运行 _my-webservice_
+
+```bash
+cd my-webservice
+mvn clean package
+MODEL_PACKAGE_NAME=$MODEL_PACKAGE_NAME docker compose up --build --force-recreate
+```
```
Comment (QubitPi) [inline docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/intro.mdx:133]:
```bash
cd my-webservice
mvn clean package
MODEL_PACKAGE_NAME=com.qubitpi.ws.jersey.template.models docker compose up --build --force-recreate

:::info

`com.qubitpi.ws.jersey.template.models` 是之前提到的[数据模型包][jersey-webservice-template-jpa-data-models]名称

:::
```

This is a documentation bug. I've corrected English version of it. Please update translation like this. Thanks

File: docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/intro.mdx [hunk 18]
```diff
@@ -0,0 +1,296 @@
...
+
+写入数据
+------------
```
Comment (QubitPi) [inline docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/intro.mdx:136]:
```
写入数据
--------
```

File: docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/intro.mdx [hunk 19]
```diff
@@ -0,0 +1,296 @@
...
+            title
+          }
+        }
+      }
+    }
+    ```
+  </TabItem>
+</Tabs>
+
+当我们运行这个 cURL 调用时，应该会看到返回很多 JSON，这就是我们新插入的对象！
```
Comment (QubitPi) [inline docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/intro.mdx:175]: `插入的对象` -> `创建的数据对象`

File: docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/intro.mdx [hunk 20]
```diff
@@ -0,0 +1,296 @@
...
+    ```
+  </TabItem>
+</Tabs>
+
+### 查看我们的数据
+
+<Tabs>
+  <TabItem value="browser" label="Web Browser" default>
+    <!-- markdown-link-check-disable -->
+    只需打开最喜欢的浏览器并访问 **http://localhost:8080/v1/data/book**
```
Comment (QubitPi) [inline docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/intro.mdx:216]: `只需打开最喜欢的浏览器并访问` -> `只需打开任意浏览器并访问`

File: docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/intro.mdx [hunk 21]
```diff
@@ -0,0 +1,296 @@
...
+    <!-- markdown-link-check-enable -->
+
+:::tip
+
+建议使用一些[JSON格式化](https://chromewebstore.google.com/detail/json-formatter/bcjindcccaagfpapjjmafapmmgkkhgoa)浏览器
+扩展来查看结果以获得更好的查看体验
```
Comment (QubitPi) [inline docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/intro.mdx:222]: `以获得更好的查看体验` -> `，数据内容更加易读`

File: docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/intro.mdx [hunk 22]
```diff
@@ -0,0 +1,296 @@
...
+            title
+          }
+        }
+      }
+    }
+    ```
+  </TabItem>
+</Tabs>
+
+### 修改数据
```
Comment (QubitPi) [inline docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/intro.mdx:256]: 更新数据

File: docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/intro.mdx [hunk 23]
```diff
@@ -0,0 +1,296 @@
...
+
+请注意，我们创建它时并没有设置新书记录的任何属性。更新数据来帮助用户与添加新数据一样简单。
```
Comment (QubitPi) [inline docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/intro.mdx:258]: `更新数据来帮助用户与添加新数据一样简单` -> `用户更新数据和添加新数据一样简单`

File: docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/intro.mdx [hunk 24]
```diff
@@ -0,0 +1,296 @@
...
+使用以下 cURL 调用更新我们的模型。
```
Comment (QubitPi) [inline docs/i18n/zh-cn/docusaurus-plugin-content-docs/current/intro.mdx:259]: `我们的模型` -> `我们的数据`

**Review by QubitPi — CHANGES_REQUESTED** [review]
Overall, pretty high-quality translation 👍🏻 Please see the review comments below.

**Comment by TristeOak** [comment]
Thank you for the review comments! I have corrected the translation and also looked into the shared links.

**Comment by QubitPi** [comment]
Awesome. I'll take a look today. Thanks

**Comment by QubitPi** [comment]
Merged. Thank you, @TristeOak 

If you are interested in more translation, I'm updating the https://github.com/paion-data/astraios/issues/55 in a moment to include more pages to translate

If you are instead interested in Webservice development using Java and [Groovy Spock](https://spockframework.org/), I'm opening up more issues and you can come back in a few days to see if any issue interests you

**Comment by TristeOak** [comment]
Thank you for merging my PR and for the information, @QubitPi 

I'm very interested in doing more translations and will continue to work on the documents according to #55. 
 
Regarding webservice development, I'm not certain of my capabilities in that area, but I'm willing to try and learn more about the project. Then, I could start with some basic tasks and gradually take on more complex issues.

**Pull request merged**